src/ValueObject/DateRangeInFuture.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\ValueObject;
+
+use App\Exceptions\InvalidDateRangeException;
+use App\Exceptions\PassedDateException;
+
+final class DateRangeInFuture
+{
+    private $startAt;
+    private $endAt;
+
+    public function __construct(\DateTimeInterface $startAt, \DateTimeInterface $endAt)
+    {
+        if ($startAt > $endAt) {
+            throw new InvalidDateRangeException(sprintf('start At %s cannot be higher than end At %s',
+                $startAt->format('Y-m-d'),
+                $endAt->format('Y-m-d')
+            ));
+        }
+
+        if ($startAt < (new \DateTime())) {
+            throw new PassedDateException(sprintf('The date %s cannot be in the past', $startAt->format('Y-m-d')));
```

Comment:
When setting a date in the past, I get this error as a Symfony error, it should be catched and displayed in a form error